### PR TITLE
set up

### PR DESCRIPTION
**Pull request by abbehjelm**
merge bitch

### CODE REVIEW

File: assignments/assignment1/craps/Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#
+# Makefile for lab1 in Process Oriented Programming
+#
+# Course: Operating Systems and Multicore Programming - OSM
+# Lab assignment 1: Game of luck
+#
+# Author: Nikos Nikoleris <nikos.nikoleris@it.uu.se>
+#
+
+# change to y to enable debugging support
```
Comment (callesebbe) [inline assignments/assignment1/craps/Makefile]: What do you mean by y here?

File: assignments/assignment1/craps/common.c [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+/* These flags control the termination of the main loop and indicate the winner. */
+volatile sig_atomic_t winner = 0;
+/* TODO: Change this to 0 to make the children spin in the for loop before they
+   receive the SIGUSR2 signal */
+volatile sig_atomic_t results = 1;
+
+/**
+ * end_handler - handle the SIGUSR2 signal, the player will receive
+ * this signal when the game ends
+ * @signum: the signal that triggered this handler
```
Comment (callesebbe) [inline assignments/assignment1/craps/common.c]: huh?

File: assignments/assignment1/craps/Makefile (hunk 1 above)
Comment (abbehjelm) [inline assignments/assignment1/craps/Makefile]: gimme a break mate

File: assignments/assignment1/craps/common.c (hunk 2 above)
Comment (abbehjelm) [inline assignments/assignment1/craps/common.c]: shfu

**Pull request merged**